### DISCUSSION TITLE
Gleylancer (Columbus Circle Reprint) problem

### DISCUSSION THREAD

**Discussion by Real-Dampfwalze** [sanni/cartreader#621] [Need help]
Tried to dump my Gleylancer reprint and it's recognized as Gleylancer, but after reading I receive a checksum error (cleaned the cartridge several times and the checksum and CRC32 is always the same, so I don't think it is a contact problem).
Rom tools show garbage in the header and the bin file doesn't work on any emulator. Fixing header checksum did also not work.
Is it possible that the cart reader uses wrong database informations? (for the old Gleylancer cartridge)
Anyone else got the same problem?
Is there a chance to get this fixed?

-----------------------------------------------------------------------------
Cart Info
 
Name: GLEYLANCER
Size: 8 MBit
ChkS: 756B
Sram: None
 

[+] Read Rom

Saving to MD/ROM/GLEYLANCER/10/...
[*******************]
Internal checksum...Error
838D

CRC32... 72A2BFE4 -> Not found

**Comment by sanni** [comment]
Do you have a picture of the PCB? 

They probably use 3V flash chips since 5V variants are not produced anymore and therefore have voltage level translator ICs on the PCB that might have an enable pin that is not set by the current Cart Reader code. Just a guess though.

I found these images of Rhythm Land 16 from the same company but I can't read what's written on the small chips:

![OXA0p0G](https://user-images.githubusercontent.com/19169494/200932460-83293055-6829-488d-b04b-38465b194daa.jpg)

![lwAZzf3](https://user-images.githubusercontent.com/19169494/200932437-5aca6da3-1d55-4db9-acc9-b63d78c940c1.jpg)

Source: https://www-elotrolado-net.translate.goog/hilo_reediciones-oficiales-pcb-de-juegos-de-mega-drive-por-columbus-circle_2340684?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-US&_x_tr_pto=wapp

**Comment by Real-Dampfwalze** [comment]
Opened the cartridge. The back label has been destroyed by my attempt to open it.
Would be great if the sacrifice will help with my problem and other owners of the game. :)
 
![Gleylancer](https://user-images.githubusercontent.com/106544997/200945593-1959eb50-3a4d-454c-be23-3d0271c758fc.jpg)

**Comment by sanni** [comment]
Could you also post a picture of the backside? Preferably with the same perspective so the two pictures can be overlayed. 
Are the markings on the big chip legible?

And do you have a multimeter to confirm that the pin circled in red goes to the gold edge connector circled in blue?

![wsfavvvd](https://user-images.githubusercontent.com/19169494/201033329-5743022a-d0da-485f-88c9-9bfa67810d24.png)

**Comment by Real-Dampfwalze** [comment]
Checked the connection with a continuity checker. There is a connection but with a resistance (the beep sound is deep, meaning there is no direct connection).
Scanned the PCBs so that you have the exact same dimension. Only problem, the scanners focus is very narrow. Hope you can use it. If not, take my first picture, cut it and resize the front and back picture to the same dimensions. :)
![Gleylancer pcb front](https://user-images.githubusercontent.com/106544997/201091988-a1acd2ba-5a90-47da-996a-1c7e3d927032.jpg)
![Gleylancer pcb back](https://user-images.githubusercontent.com/106544997/201092143-ca298b56-bb21-41a0-ab63-786bc2ba387b.jpg)

**Comment by sanni** [comment]
Resized them, so they can be overlayed:

![bottom](https://user-images.githubusercontent.com/19169494/201112181-1ceb38cf-605d-43f6-bdb0-a5691e6783f1.jpg)
![top](https://user-images.githubusercontent.com/19169494/201112187-b153f58b-b418-4934-b1a5-373dac0a409c.jpg)
![top2](https://user-images.githubusercontent.com/19169494/201112192-5aa4495e-ee77-4329-8f17-2ffbd5b90530.jpg)
